docs/aurora/data-management/daos/daos-overview.md

```diff
@@ -202,77 +372,87 @@ Each DAOS server nodes is based on the Intel Coyote Pass platform.
-As such, it might be beneficial to convert the code to write each checkpoint/time step into a HDF Group and then a single HDF file can be used for the entire campaign.
-This solution is more DAOS specific, as it will be functionally compatible on any system, however a traditinoal PFS may lose the entire contents of the file if a failure occurs during write while DAOS will be resilent to those failures and rollback to a previous good version.
-
-## DFS
-DFS is the user level API for DAOS.
-This API is very similar to POSIX but still has many differences that would require code changes to utilize DFS directly.
-The DFS API can provide the best overall performance for any scenario other than workloads which benefit from caching.
+## Darshan profiler for DAOS 
+
+Currently, you need to install your own local darshan-daos profiler
+You need to use DFS mode (3) or Posix with interception library to profile
+
+```bash
+module use /soft/modulefiles
+module load daos
+module list
+git clone https://github.com/darshan-hpc/darshan.git
+git checkout snyder/dev-daos-module-3.4
+./prepare.sh 
+mkdir /home/kaushikvelusamy/soft/profilers/darshan-daos/darshan-install
+
+./configure --prefix=/home/kaushikvelusamy/soft/profilers/darshan-daos/darshan-install  \
+            --with-log-path=/home/kaushikvelusamy/soft/profilers/darshan-daos/darshan-logs \
+            --with-jobid-env=PBS_JOBID \
+            CC=mpicc --enable-daos-mod
+
+make && make install 
+
+chmod 755 ~/soft/profilers/darshan-daos/darshan/darshan-install/darshan-mk-log-dirs.pl
+mkdir /home/kaushikvelusamy/soft/profilers/darshan-daos/darshan-logs
+cd /home/kaushikvelusamy/soft/profilers/darshan-daos/darshan-logs
+~/soft/profilers/darshan-daos/darshan/darshan-install/darshan-mk-log-dirs.pl
+~/soft/profilers/darshan-daos/darshan-install/bin/darshan-config  --log-path
+
+```
+
+Preload darshan first then daos interception library
+
+```
+mpiexec --env LD_PRELOAD=~/soft/profilers/darshan-daos/darshan-install/lib/libdarshan.so:/usr/lib64/libpil4dfs.so   
+        -np 32 -ppn 16  --no-vni -genvall \
+        ior -a DFS  --dfs.pool=datascience_ops --dfs.cont=ior_test1   \
+            -i 5 -t 16M -b 2048M  -w  -r -C -e    -c  -v -o /ior_2.dat 
+```
+
+
+install darshan-util from laptop
+
+
+```bash
+
+conda info –envs
+conda activate env-non-mac-darshan-temp
+/Users/kvelusamy/Desktop/tools/spack/share/spack/setup-env.sh 
+
+spack install darshan darshan-util
+export DYLD_FALLBACK_LIBRARY_PATH=/Users/kvelusamy/Desktop/tools/spack/opt/spack/darwin-ventura-m1/apple-clang-14.0.3/darshan-util-3.4.4-od752jyfljrrey3d4gjeypdcppho42k2/lib/:$DYLD_FALLBACK_LIBRARY_PATH
+
+darshan-parser ~/Downloads/kaushikv_ior_id917110-44437_10-23-55830-632270104473632905_1.darshan 
+python3 -m darshan summary ~/Downloads/kaushikv_ior_id917110-44437_10-23-55830-632270104473632905_1.darshan #coming soon
+
+```
+
+## Cluster Size
+
```

Comment:
Should this be "Number of available DAOS servers"

Reply:
updated.